homework-submission/homeworkweek4/react-hyf/react-hyf-client/src/components/Main/Main.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Main.css';
+
+const Main = ({ elements }) => {
+  const { Home, ...weeks } = elements;
+
+  return (
+    <main>
+      <ul className="weeks">
+        {Object.keys(weeks).map(key => (
+          <li key={key}>
+            <Link className="weeks-item" to={weeks[key].path}>
+              {key}:{weeks[key].description}
+            </Link>
+          </li>
```

Comment:
I see this ```     <li key={key}>
            <Link className="weeks-item" to={weeks[key].path}>
              {key}:{weeks[key].description}
            </Link>
          </li>``` in mulitple components. Could be a idea to make a component out of it. Also Main.js and NavBar.js are almost doing the same. You can create a component and use it inside the both of them.